### PR TITLE
feat: Add Sort By Filters

### LINKED ISSUE #1666: Add Sort By Filter
**Describe the feature you'd like**
Add Sort By Filters in Filter fragment with name, date, price, etc

**Would you like to work on the issue?**
- [x] Yes

### PR DESCRIPTION
**Pull request by aggarwalpulkit596**
Fixes #1666 

Changes: Added Sort By Name, Date Filters in SearchFiltersFragment

Screenshots for the change:
videotogif_2019 04 25_22 20 37

### CODE REVIEW

**Comment by aggarwalpulkit596** [comment]
@nikit19 @iamareebjamal please review

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -313,5 +313,9 @@
     <string name="filter_set">Filter Done</string>
     <string name="filter">Filter Search</string>
     <string name="free_stuff">Free stuff only</string>
+    <string name="sort_by_name">name</string>
+    <string name="sort_by_date">starts-at</string>
+    <string name="sort_by">Sort By</string>
+
```
Comment (nikit19) [inline app/src/main/res/values/strings.xml]: extra

File: app/src/main/java/org/fossasia/openevent/general/search/SearchFilterFragment.kt [hunk 2]
```diff
@@ -8,35 +8,55 @@ import android.view.MenuInflater
...
 import org.fossasia.openevent.general.utils.Utils.getAnimFade
 import org.fossasia.openevent.general.utils.Utils.setToolbar
 
 class SearchFilterFragment : Fragment() {
-
     private lateinit var rootView: View
     private var isFreeStuffChecked = false
     private val safeArgs: SearchFilterFragmentArgs by navArgs()
-
+    private var sortBy: String = "name"
```
Comment (nikit19) [inline app/src/main/java/org/fossasia/openevent/general/search/SearchFilterFragment.kt]: I guess this can be `lateinit`

**Comment by aggarwalpulkit596** [comment]
@nikit19 updated

**Comment by nikit19** [comment]
Build failed

**Comment by aggarwalpulkit596** [comment]
@nikit19 fixing spotless error 😩

**Comment by aggarwalpulkit596** [comment]
@nikit19 @iamareebjamal updated

**Comment by aggarwalpulkit596** [comment]
@nikit19 anything left out ?

**Pull request merged**